### ISSUE TITLE
Accessibility: Nav Menu Should Have Labels

### ISSUE THREAD

**Issue by DeadSuperHero** [EmissarySocial/emissary#545]
<img width="1126" height="280" alt="Image" src="https://github.com/user-attachments/assets/10f66848-6e22-42e5-940f-aaa97b28dd10" />

The Nav menu looks pretty clean nowadays, but it lacks a11y support. Additionally: while the icon-only look is visually great, it's not necessarily clear which button goes to which part of the app.

A couple things we can do to improve this:
* Use popovers that appear when links are hovered
* Add some labels in the HTML structure for screen readers
* Redesign the menu

**Comment by benpate** [comment]
Agreed. Will do.

**Comment by benpate** [comment]
I've added ARIA labels to these, which should take care of screen readers.

I think the icons alone should work well enough for people using regular browsers, but we can look at more refinements after we get more user feedback on this design.
